en/frames_boat/bigstorm.md

```diff
@@ -0,0 +1,52 @@
+# Big Storm Boat build
+
+[*Big Storm*](https://www.galaxus.ch/de/s5/product/amewi-big-storm-rennboot-rc-boot-13369624) is a COTS (Commercial Off The Shelf) racing RC boat with a rudder, single propeller thruster and a water cooled ESC.
+
+![Big Storm Shop image](../../assets/airframes/boat/bigstorm/bigstorm_shop.jpg)
+
+## Spec
+
+- Max speed : 65 km/h
+- Length : 740 mm
+- Width : 180 mm
+- Weight : 1200 g
+- Body : ABS Plastic
+- Drive shaft : 4mm flexible shaft
+- 2815 Brushless outrunner motor, water-cooled
+- 37 gram Steering servo
+- 60A Water cooled ESC (upto 4S voltage)
+- Recommended battery : 2 x 2S LiPo 7.4V 3000 - 4000mAh 40C
+
+## Modification process
```

Comment:
We use word capitalization of headings "Modification Process" - applies everywhere.